server/stream.go

```diff
@@ -3447,8 +3453,10 @@ func (mset *stream) processInboundSourceMsg(si *sourceInfo, m *inMsg) bool {
 	hdr, msg := m.hdr, m.msg
 
 	// If we are daisy chained here make sure to remove the original one.
+	// If the TTL header is set, remove it.
```

Comment:
Let's rework this logic to not remove but ignore.

Reply:
Is this wise? If we have to rebuild the mirroring/sourcing stream, how do we know whether or not to recover the TTLs? Do we just never recover the TTLs if there are mirrors/sources present?